src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -554,6 +567,11 @@ def load_arguments(self, _):
                    help='Name of the server to create the copy in.'
                    ' If unspecified, defaults to the origin server.')
 
+        c.argument('storage_account_type',
```

Comment:
allowed value is in ['LRS', 'ZRS‘, 'GRS']? What about RA-GRS, RA-GZRS, GZRS?

Reply:
Backup storage uses LRS/ZRS/GRS